File: students/students/src/MON.SPA/src/models/lodModels/lodFirstGradeEvaluationResultModel.js
export class LodFirstGradeEvaluationResultModel {
  constructor(obj = {}) {
    this.id = obj.id || null;
    this.personId = obj.personId || null;
    this.schoolYear = obj.schoolYear || null;
    this.studentClassId = obj.studentClassId || null;
    this.gradeResult = obj.gradeResult || null;
  }
}

File: students/students/src/MON.Models/StudentModels/Update/LodFirstGradeEvaluationResultModel.cs
using System.ComponentModel.DataAnnotations;

namespace MON.Models.StudentModels.Update
{
    public class LodFirstGradeEvaluationResultModel
    {
        [Required]
        public int PersonId { get; set; }
        [Required]
        public short SchoolYear { get; set; }
        [Required]
        public int GradeResult { get; set; }
        public int? StudentClassId { get; set; }
    }
}
